tests/rbd_mirror/test_rbd_mirror_image_features.py

```diff
@@ -0,0 +1,211 @@
+import time
+
+from tests.rbd.rbd_utils import Rbd
+from tests.rbd_mirror.rbd_mirror_utils import rbd_mirror_config
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def test_journal_image_feature(rbd_mirror, pool_type, **kw):
+    """Method to validate change of image feature in primary
+    got reflected onto secondary in Journal based RBD mirroring.
+    Args:
+        rbd_mirror: Object
+        pool_type: Replicated or EC pool
+        **kw: test data
+    Returns:
+        int: The return value. 0 for success, 1 otherwise
+    """
+
+    try:
+        log.info(
+            "Running RBD journal based mirroring test case for image features reflection"
+        )
+        config = kw.get("config")
+        mirror1 = rbd_mirror.get("mirror1")
+        mirror2 = rbd_mirror.get("mirror2")
+        rbd1, rbd2 = [
+            Rbd(**kw, req_cname=cluster_name)
+            for cluster_name in kw.get("ceph_cluster_dict").keys()
+        ]
+        poolname = config[pool_type]["pool"]
+        imagename = config[pool_type]["image"]
+        imagespec = poolname + "/" + imagename
+
+        image_feature = "object-map,fast-diff"
+        features = list(image_feature.split(","))
+        # Disable image features in primary cluster
+        mirror1.image_feature_disable(imagespec=imagespec, image_feature=image_feature)
+        # wait till image gets reflected to secondary
+        time.sleep(60)
```

Comment:
please avoid hard time sleep. use 300 seconds to check feature disable ad enable.

Reply:
Done